### PR TITLE
Add console connection broken handler

### PR DESCRIPTION
**Pull request by kiya956**

### CODE REVIEW

File: sanity/agent/console.py [hunk 1]
```diff
@@ -19,14 +42,28 @@ class Console:
...
+
+    def close(self):
+        """close port"""
+        self.con.close()
 
+    def connect(self, com_port, brate):
+        """
+        This function is for handling serial console connection
+        """
+        attempt = 0
```
Comment (tsunghanliu) [inline sanity/agent/console.py:58]: `attempt` is reset in `broken_handler()`, but is also reset here.
Will this cause unexpected attempts when called by `broken_handler()`?

**Review by tsunghanliu — COMMENTED** [review]
please see inline comments

File: sanity/agent/console.py (hunk 1 above)
Comment (kiya956) [reply sanity/agent/console.py:58]: broken_handler is not a function be call but a decorator. And attempt is local variable will not affect cross functions

File: sanity/agent/console.py [hunk 2]
```diff
@@ -6,11 +6,34 @@
...
+    return wrapper
+
+
 class Console:
     """handle send and receive through console to target"""
 
     con = None
     device_uname = ""
+    com_port = ""
+    brate = 115200
```
[the remark this answers is not in the document]
Comment (kiya956) [reply sanity/agent/console.py]: done

File: sanity/agent/console.py (hunk 1 above)
Comment (tsunghanliu) [reply sanity/agent/console.py:57]: Thanks! I didn't notice the variable is local.

**Review by tsunghanliu — APPROVED** [review]

**Pull request merged**